frontend/src/modules/entitydetails/components/Screenshots.tsx

```diff
@@ -15,37 +16,26 @@ type Props = {
  * This component looks at all URLs to an image (either .png or .jpg) in a
  * source string and then shows a miniature of those images.
  */
-export default class Screenshots extends React.Component<Props> {
-    openLightbox: (image: string) => () => any = (image: string) => {
-        return () => this.props.openLightbox(image);
-    };
+export default function Screenshots(props: Props) {
+    const { locale, source } = props;
+    const dispatch = useAppDispatch();
 
-    getImages(): Array<React.ReactNode> {
-        const { locale, source } = this.props;
+    const images = getImageURLs(source, locale);
 
-        const images = getImageURLs(source, locale);
-        return images.map((urlWithLocale, i) => {
-            return (
+    if (images.length === 0) {
+        return null;
+    }
+
+    return (
+        <div className='screenshots'>
+            {images.map((urlWithLocale, i) => (
                 <img
                     src={urlWithLocale}
                     alt=''
                     key={i}
```

Comment:
```suggestion
            {images.map((urlWithLocale) => (
                <img
                    src={urlWithLocale}
                    alt=''
                    key={urlWithLocale}
```

Nitpick: Am I right in presuming that `urlWithLocale` will not be repeated within `images`? If so, it's a much better unique identifer than `i`.

Reply:
I agree that the index is probably the worst-practice choice here :-/

I guess it's safe to assume that folks are not going to have a comment with multiple references to the same image in there.